wordpress-mobile/gutenberg-mobile
# Fix heading block shadow on links.

Fixes #2507 
Fixes #1654 

<img src="https://user-images.githubusercontent.com/651601/88405974-ead16980-cdc7-11ea-8bef-a136148a699a.png" width=300 />


To test:
 - Make sure the device/sim is running in dark mode.
 - Start the demo app
 - Add or select a heading block
 - Write some text
 - Apply a link to the text 
 - Apply strong to the text.
 - Check the text looks correct

PR submission checklist:
